tests/e2e/scripts/fast_test.sh

```diff
@@ -31,7 +31,10 @@ cd ../
 cat >config.json<<END
 {
         "image_url": ["nginx", "hello-world"],
-        "k8smasterforkubeedge":"http://$MASTER_IP:12418"
+        "k8smasterforkubeedge":"http://$MASTER_IP:12418",
+         "dockerhubusername":"user",
+         "dockerhubpassword":"password",
```

Comment:
Is this field useful ? Do we need dockerHub login details to pull public images ?

Reply:
There is a docker push involved , so these are requied.

Reply:
Are these the correct username password ?

Reply:
Updated from config